eox_nelp/api_clients/pearson_rti.py

```diff
@@ -62,7 +62,6 @@ def ping(self, payload):
 
         return result
 
-    @audit_method(action="Import Candidate Demographics")
```

Comment:
So basicly, the problem is that could be serialized the error because is a function attr of the  PearsonRTIApiClient class.?

Reply:
eox-audit takes as parameter the self declaration, in this case self is an instance of PearsonRTIApiClient and that class is not serializable, the inner method removes the self from the eox-audit arguments